### ISSUE TITLE
M1 Macbook Air Display 0 brightness, not changing even when adjusted using brightness keys nor Monitor Control sloders

### ISSUE THREAD

**Issue by imiiiiiiiiil** [MonitorControl/MonitorControl#1160]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

M1 Macbook Air Display 0 brightness, not changing even when adjusted using brightness keys nor Monitor Control sliders

Now, even when I disconnect the external display, the Macbook’s brightness is still on 0.

### Expected behavior

Brightness sliders should be able to control Macbook’s built-in display.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model: M1 Macbook Air
- MonitorControl version: 4
- Monitor(s): AOC 2790 VQ
- Monitor Cable(s)/Connection(s): HDMI, hyperdrive dongle
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Does the brightness change work without MonitorControl running?

**Comment by imiiiiiiiiil** [comment]
> Does the brightness change work without MonitorControl running?

Hi! The internal monitor/built-in display brightness has stopped working. 

This is how I discovered the problem:

Attached is a photo of my current set up.

The Macbook is on a rack with wheels, attached to an external display. Since I am only using the external display, I set the brightness of the built-in display to 0 (using Monitor Control). I continue to to work through the day.

A few hours later, after I'm done with work, I disconnect the HDMI from the laptop, and turn up the brightness on the built-in display, and the brightness just doesn't adjust.

Rebooting, not even restoring a Time Machine backup worked.

Any help would be appreciated.

![IMG_1105](https://user-images.githubusercontent.com/102204028/188258179-6e941c97-2bcf-4047-9ba9-17bd3c236299.jpg)

**Comment by waydabber** [comment]
If you move the macbook to a dark room and turn it on/off, do you see any backlight coming up? If your macbook display backlight generally does not work (even on boot), it might be that the backlight is actually faulty and the laptop might need servicing.

**Comment by imiiiiiiiiil** [comment]
> If you move the macbook to a dark room and turn it on/off, do you see any backlight coming up? If your macbook display backlight generally does not work (even on boot), it might be that the backlight is actually faulty and the laptop might need servicing.

Unfortunately, no backlight coming up. I can see that the screen is alive by pointing a flashlight directly onto the screen on a dark room —— but other than that, none. 

By any chance, maybe the Monitor Control is overriding some system/hardware settings?

**Comment by waydabber** [comment]
No, MonitorControl simply can't do that. If control of the internal backlight is enabled, all it does is it calls Apple's API to control the backlight, just like if you used the normal macOS brightness slider for the internal display. :(

**Comment by waydabber** [comment]
I'll convert this into a discussion, maybe somebody in the same shoes will have some suggestion. On Intel you could try SMC reset and stuff like that, but on M1 there is no such things, a simple reboot or power-cycle should handle these things.
